template/express-rest-api/package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "name": "<%- serviceName %>",
```

Comment:
Should we put a `@seek/` scope out of paranoia?

I've seen us pay out bounties for "dependency confusion" even though I tend to disagree with the premise.